cg/constants/constants.py

```diff
@@ -258,6 +258,13 @@ class MicrosaltAppTags(StrEnum):
     PREP_CATEGORY: str = "mic"
 
 
+MICROBIAL_APP_TAGS = [
+    MicrosaltAppTags.MWRNXTR003,
+    MicrosaltAppTags.MWXNXTR003,
+    MicrosaltAppTags.VWGNXTR001,
+]
```

Comment:
Should these not be fetched from statusdb? Otherwise we need to update this whenever an app tag is archived or introduced?

Reply:
Good point, I will take a look at this

Reply:
fixed